src/common.js

```diff
@@ -53,6 +53,20 @@ function ShellString(str) {
 }
 exports.ShellString = ShellString;
 
+// Return the home directory in a platform-agnostic way, with consideration for
+// older versions of node
+function getUserHome() {
```

Comment:
In terms of implementation, this should be very similar. I opted to just code it myself because it cuts out a dependency and it was only a few lines of code. I think it's easier to keep small snippets in the code base.